docs/testplan/pac/Port_Access_Control.md

```diff
@@ -0,0 +1,562 @@
+
+
+### **3.3 Reboot and Trigger Test Cases**
+
+### 3.3.1 Verify Client authentication after reboot
+
+| **Test ID**    | **PAC_FUNC_TRIGGER_001**                 |
+| -------------- | :--------------------------------------- |
+| **Test Name**  | **Verify Client authentication after reboot**|
+| **Test Setup** | **Topology1**                            |
+| **Type**       | **Functional**                           |
+| **Steps**      | 1. Define RADIUS server<br/>2. Enable 802.1x/MAB globally and at interface level<br/>3. Authenticate to the DUT with multiple 802.1x clients and MAB clients<br/>4. Do config save and "reboot"<br/>5. 802.1x/MAB clients are removed and authenticated again.<br/>6. Verify that 802.1x/MAB clients authentication is successful<br/> |
+
+
+### 3.3.2  Verify Client authentication after warmboot
+
+| **Test ID**    | **PAC_FUNC_TRIGGER_002**                 |
+| -------------- | :--------------------------------------- |
+| **Test Name**  | **Verify Client authentication after warmboot**|
+| **Test Setup** | **Topology1**                            |
+| **Type**       | **Functional**                           |
+| **Steps**      | 1. Define RADIUS server<br/>2. Enable 802.1x/MAB globally and at interface level<br/>3. Authenticate to the DUT with multiple 802.1x clients and MAB clients<br/>4. Do config save and warmboot<br/>5. 802.1x/MAB clients are removed and authenticated again.<br/>6. Verify that 802.1x/MAB clients authentication is successful<br/> |
+
+
+### **3.4 Scale Test Cases**
+
+### 3.4.1 Verify 128 max supported 802.1x clients on DUT.
+
+| **Test ID**    | **PAC_SCAL_001**                         |
+| -------------- | :--------------------------------------- |
+| **Test Name**  | **Verify 128 max supported 802.1x clients on DUT.**|
+| **Test Setup** | **Topology1**                            |
+| **Type**       | **Functional**                           |
+| **Steps**      | 1. Enable 802.1x authentication globally and at interface level on multiple ports<br/>2. configure-authentication in multi-auth mode<br/>3. Try authentication 128 clients across multiple interfaces(at least 16 clients from one or two interfaces to cover max clients per port) and verify 128 802.1x authenticated clients<br/>4. Verify FDB entries for all 128 clients<br/>5. Verify all client traffic gets allowed after authentication<br/>6. Logoff all the 128 clients and verify clients move to unauthorized state and gets blocked from accessing the server.<br/>7. Reinitiate the clients and check all are authenticated successfully.<br/>8. Log off the clients again and check all are cleared.<br/> |
+
+
+### 3.4.2 Verify 128 max supported MAB clients on DUT.
```

Comment:
Please include the mix of 802.1x and MAB scenario, meaning the total number of 802.1x and MAB clients is limited to 128.

Reply:
Added new test case